Tests/Functional/SwiftPMXCTestHelperTests.swift

```diff
@@ -22,10 +22,11 @@ class SwiftPMXCTestHelperTests: XCTestCase {
             // Run swift-test on package.
             XCTAssertSwiftTest(prefix)
             // Expected output dictionary.
-            let testCases = ["name": "All Tests", "tests": [[
+            let testCases = ["name": "All Tests", "tests": [["name" : "SwiftPMXCTestHelperTests.xctest",
```

Comment:
Is `SWIFTPMXCTestHelperTests.xctest` really the right name? I wouldn't have expected the trailing `.xctest`.